db/schema.rb

```diff
@@ -10,10 +10,9 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2022_03_01_182323) do
+ActiveRecord::Schema.define(version: 2024_01_26_170854) do
 
   # These are extensions that must be enabled in order to support this database
-  enable_extension "pg_stat_statements"
```

Comment:
Not sure if having this extension removed will cause issues.  Probably best to keep it in.